gcn/notices/chime/detection.example.json

```
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/chime/frb.schema.json",
+  "alert_type": "detection",
+  "trigger_time": "2024-09-18T07:19:10.765268Z",
+  "trigger_time_error": 0.007864319719374001,
+  "id": "427325191",
```

```suggestion
  "id": ["427325191"],
```
This property comes from https://github.com/nasa-gcn/gcn-schema/blob/main/gcn/notices/core/Event.schema.json, and is defined as an array of numbers or strings